pkgs/development/coq-modules/serapi/default.nix

```diff
@@ -27,7 +28,11 @@ in
   pname = "serapi";
   inherit version release;
 
-  defaultVersion =  with versions; switch coq.version [
+  defaultVersion =  with versions;
+    if isEq "8.14" coq.version && isLt "4.12" coq.ocamlPackages.ocaml.version
+    then "8.14.0+0.14.0"
+    else
```

Comment:
@vbgl why don't you integrate this to the switch below?

Reply:
Not sure how to do it. But your suggestion is welcome.

Reply:
I’ve just proposed an other version. Thanks.

Reply:
Looks better indeed